.setup/pip/system_requirements.txt

```diff
@@ -0,0 +1,50 @@
+# Requirements.txt for the entire project
+
+# Migrator module depends on:
+# docker & sqlalchemy
+
+# Auto-grader module depends on:
+# sqlalchemy & paramiko & docker & python-dateutil & submitty_util (internal)
+
+# Submitty util module depends on:
+# jsonschema & jsonref & pytz & tzlocal
+pytz==2021.1 # Submitty-util specific.
+
+# Submodule dependency last updated: May 17 2021.
+
+
+python-pam==1.8.4
+PyYAML==3.12
+psycopg2-binary==2.8.6
+sqlalchemy==1.3.24
+pylint==2.8.1
+psutil==5.8.0
+python-dateutil==2.8.1
+watchdog==2.0.3
+xlsx2csv==0.7.8
+pause==0.3
+paramiko==2.7.2
+tzlocal==2.1
+PyPDF2==1.26.0
+distro==1.5.0
+jsonschema==3.2.0
+jsonref==0.2
+docker==5.0.0
+
+# For Lichen / Plagiarism Detection
+parso==0.8.2
+
+# Python3 implementation of python-clang bindings (may not work < 6.0)
+clang==11.0
+
+#python libraries for QR bulk upload
+pyzbar==0.1.8
+pdf2image==1.14.0
+opencv-python==4.5.1.48
```

Comment:
We require an older version (`opencv-python==3.4.9.33`) so that it'll install cleanly on older versions of Ubuntu (18.04).

Reply:
@MasterOdin I am not very familiar with this package, how should I write the note?
Is `To pin this package to version 3.4.9.33 allows a clean install on older versions of Ubuntu (18.04)` sufficient?

Reply:
Resolved in d810dc9, noted with:
```
# The use of this older version of `opencv-python` is deliberate
# This allows a clean install on older versions of Ubuntu (18.04).
```